### PR TITLE
Add features list for KubeKey

### PR DESCRIPTION
**Pull request by pixiake**
Signed-off-by: feng <guofeng@yunify.com>

### What type of PR is this?
/kind documentation

### What this PR does / why we need it:
Add a complete list of  features for KubeKey
### Which issue(s) this PR fixes:

Fixes #

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: docs/features.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Features
+### Cluster lifecycle management
```
Comment (Patrick-LuoYu) [inline docs/features.md:2]: Cluster Lifecycle Management

File: docs/registry.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+kind: Cluster
+metadata:
+  name: sample
+spec:
+  hosts:
+  - {name: node1, address: 192.168.6.6, internalAddress: 192.168.6.6, password: Qcloud@123}
+  roleGroups:
+    etcd:
+    - node1
+    master:
```
[the remark this answers is not in the document]
Comment (pixiake) [reply docs/registry.md]: done

**Pull request merged**